ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -188,7 +188,7 @@
 				TargetAttributes = {
 					97C146ED1CF9000F007C117D = {
 						CreatedOnToolsVersion = 7.3.1;
-						DevelopmentTeam = FC2EL78J4P;
+						DevelopmentTeam = W2B7PMH9SQ;
```

Comment:
What it means? Would it affect last version released into testflight?

Reply:
Every developer needs to change the team to their own team so they can install on device

Until we have a shared team for app store.

Reply:
Yeah, I changed the team to my team so I can install the app on my iOS device.